Source/WebKit/WPEPlatform/wpe/drm/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ target_compile_options(WPEPlatformDRM PRIVATE ${DEVELOPER_MODE_CXX_FLAGS})
 target_include_directories(WPEPlatformDRM PRIVATE ${WPEPlatformDRM_PRIVATE_INCLUDE_DIRECTORIES})
 target_include_directories(WPEPlatformDRM SYSTEM PRIVATE ${WPEPlatformDRM_SYSTEM_INCLUDE_DIRECTORIES})
 target_link_libraries(WPEPlatformDRM ${WPEPlatform_LIBRARIES} ${WPEPlatformDRM_LIBRARIES})
+add_dependencies(WPEPlatformDRM WPEPlatformGeneratedEnumTypesHeader)
```

Comment:
In theory this should not be needed, because `WPEPlatformDRM` depends on `WPEPlatform`, and `WPEPlatform` depends on `WPEPlatformGeneratedEnumTypesHeader`&mdash;in theory, at least.